src/QandA/components/AnswerItem.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import Photos from './Photos.jsx';
+import moment from 'moment';
+import axios from 'axios';
+
+class AnswerItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      helpfulness: this.props.answer.helpfulness,
+      reported: false,
+      updatedHelpfulness: false,
+      updatedReport: false
+    }
+    this.updateHelpful = this.updateHelpful.bind(this);
+    this.updateReport = this.updateReport.bind(this);
+    this.byLine = this.props.answer.answerer_name === 'Seller' ? <b>Seller</b> : <span>{this.props.answer.answerer_name}</span>;
+    this.render = this.render.bind(this);
+  }
+
+
+  updateHelpful() {
+    if(!this.state.updatedHelpful) {
+      axios.put(`/qa/answers/${this.props.answer.answer_id}/helpful`)
+      .catch((error) => {
+        console.error('Helpful Button Click error: ', error);
+      });
+      this.setState({
+        updatedHelpful :  true,
+        helpfulness: this.state.helpfulness + 1
+      });
+    }
+  }
+
+  updateReport() {
+    if (!this.state.updatedReport) {
+      axios.put(`/qa/answers/${this.props.answer.answer_id}/report`)
+      .catch((error) => {
+        console.error('Answer Report Button Error: ', error);
+      });
+      this.setState({
+        reported : true
+      });
+    }
+  }
+
+  reportView() {
+    const {reported} = this.state;
+
+    if (!reported) {
```

Comment:
how about just using the html5 attribute "disabled" for button?
`<button type="button" disabled>`